main.go

```diff
@@ -122,6 +123,29 @@ func main() {
 		os.Exit(1)
 	}
 
+	var reader io.Reader
+	// If file being formatted is being passed via stdin determine projectName via working dir
+	if filePath == "-" {
+		reader = os.Stdin
+		path, err := os.Getwd()
```

Comment:
What if we perform this option from outside of a project? 
Example:
```
cd /root && cat /user/someuser/workspace/github.com/someuser/project/some.go | goimports-reviser -file-path -
```
I think we should try to determine projectName here and in case of error should mark this option as required